helpers/timesHelper.js

```diff
@@ -0,0 +1,232 @@
+const moment = require('moment-timezone');
+const { Configuration } = require('@schul-cloud/commons');
+const logger = require('./logger');
+
+const DEFAULT_TIMEZONE = Configuration.get('I18N__DEFAULT_TIMEZONE');
+
+let schoolTimezone;
+let userTimezone;
+let userHasSchoolTimezone = true;
+
+let FORMAT = {
```

Comment:
i would place default format near I18N__DEFAULT_TIMEZONE (in configuration) or fetch these information directly from moment based on timezone or may we place both values in locale?